packages/editor/src/client/blocks/embeds/iframe.tsx

```diff
@@ -4,15 +4,15 @@ import {transformCssStringToObject} from '../../utility'
 export interface IframeEmbedProps {
   url?: string
   title?: string
-  width?: number
-  height?: number
+  width?: string
+  height?: string
   styleCustom?: string
   sandbox?: string
 }
 
 export function IframeEmbed({url, title, width, height, styleCustom, sandbox}: IframeEmbedProps) {
-  const ratio = width !== undefined && height !== undefined ? width / height : 0
-  const noRatio = !!styleCustom && ratio === 0
+  const ratio = width !== undefined && height !== undefined ? parseInt(width) / parseInt(height) : 0
```

Comment:
Since with and height are strings could you not write !width and !height or is there an other action if the values are null?

Reply:
I assume you're right. Will change this.